android/capacitor/src/main/java/com/getcapacitor/WebViewLocalServer.java

```diff
@@ -350,6 +350,28 @@ private WebResourceResponse handleProxyRequest(WebResourceRequest request, PathH
                         String base64 = Base64.encodeToString(userInfoBytes, Base64.NO_WRAP);
                         conn.setRequestProperty("Authorization", "Basic " + base64);
                     }
+
+                    /**
+                     * This should be the final step in the setup process of the HttpURLConnection object.
+                     *
+                     * Initiate the connection with follow redirects disable. It allows us
+                     * identifying any redirection response and avoid intercepting such request.
+                     * This is because returning a 200 response when a redirection response (ie. 301 or 302)
+                     * is expected could result in issues with resolving relative URLs for static assets
+                     * in the HTML.
+                     */
+                    conn.setInstanceFollowRedirects(false);
+                    conn.connect();
+                    int status = conn.getResponseCode();
+                    if (
+                        status == HttpURLConnection.HTTP_MOVED_TEMP ||
+                        status == HttpURLConnection.HTTP_MOVED_PERM ||
+                        status == HttpURLConnection.HTTP_SEE_OTHER
```

Comment:
This PR worked great for me! I'm depending on a fork now. The one thing is that I also had to add `|| status == 307` here.

Reply:
Yeah, that make sense. Though 307 is not a very common status code isn't it?

Reply:
It is not, that's why there's no constant for it in `HttpURLConnection`. But a backend we had to support happens to use it and it is standard: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/307

Thanks for adding it!